docs/interpretation.rst

```diff
@@ -28,7 +28,7 @@ Data model
       - The current resolution status. REQUIRED.
     * - diagnosis
       - :ref:`rstdiagnosis`
-      - 0..*
+      - 0..1
       - One or more diagnoses, if made
```

Comment:
What about multiple diagnoses i.e. from a blended phenotype? So, if they should be represented as two separate `Interpretation` messages the description should read `The diagnosis, if made`